packages/create-plugin/src/commands/update.command.ts

```diff
@@ -20,6 +21,11 @@ export const update = async () => {
     // -----------------------------------
     if (await confirmPrompt(TEXT.updateConfigPrompt)) {
       compileTemplateFiles(UDPATE_CONFIG.filesToOverride, getTemplateData());
+
+      if (isPrettierUsed()) {
+        prettifyFiles('.config', '.');
+      }
```

Comment:
**Nit:** we could make the call-site simpler by moving the `if (isPrettierUsed()) { ...` check into the `prettifyFiles()` function.

Reply:
I think it would also make sense to use named parameters for `prettifyFiles()`, it would make it more clear by looking at it what those strings / paths are, e.g.:

```typescript
type PrettifyFilesArgs = {
  // The path where we want to prettify files, defaults to the CWD
  targetPath?: string;
  
  // In case the `targetPath` would be set to a subdirectory, e.g. `.config/`. Defaults to `targetPath`.
  projectRoot?: string;
}

export async function prettifyFiles(options: PrettifyFilesArgs) {
  const targetPath = options.targetPath ?? process.cwd();
  const projectRoot = options.projectRoot ?? targetPath;
   
```